### ISSUE TITLE
Recreate some of Aidan's work

### ISSUE THREAD

**Issue by MichaelClerx** [ModellingWebLab/project_issues#17]
In this case study we can look at using derived measures, including visualisation and fitting, using Aidan's 2015 paper as a basis

- [ ] Add ABC-SMC method to pints
- [ ] Recreate some of Aidan's results in Pints

Once this is done we can again look into getting these things in web lab!

**Comment by MichaelClerx** [comment]
This might happen eventually, but not a priority now!
